### PR TITLE
New free plan

### PR DESCRIPTION
**Pull request by gonzaloriestra**
Related to: CartoDB/cartodb-central#2670

### CODE REVIEW

File: app/helpers/frontend_config_helper.rb [hunk 1]
```diff
@@ -82,20 +80,10 @@ def frontend_config_hash(user = current_user)
       config[:dataservices_enabled] = Cartodb.get_config(:dataservices, 'enabled')
     end
 
-    if CartoDB.account_host.present? && show_account_update_url(user)
-      config[:account_update_url] = "#{CartoDB.account_host}"\
```
Comment (alrocar) [inline app/helpers/frontend_config_helper.rb]: I see we used this config param in several places, what's the impact of this change?

Comment (gonzaloriestra) [reply app/helpers/frontend_config_helper.rb]: I couldn't find any usage... where?

Comment (alrocar) [reply app/helpers/frontend_config_helper.rb]: frontend side

Comment (alrocar) [reply app/helpers/frontend_config_helper.rb]: 🤐 it was old stuff I had locally, we don't need it anymore

**Pull request merged**